values.yaml

```
@@ -564,6 +564,8 @@ dagDeploy:
 
   serviceAccount:
     annotations: {}
+    name: ""
```

Shouldn't this be null?
```suggestion
    name: ~
```